samples/TestApplication/Pages/_ViewImports.cshtml

```diff
@@ -1 +1,2 @@
-﻿@layout MainLayout
+﻿@using System.Collections.Generic;
```

Comment:
I think I don't need this. This was part of a frantic "Why isn't this working?" 1 hour